src/store/result/types/result.types.ts

```diff
@@ -0,0 +1,3 @@
+export const FETCH_PETS_REQUEST = 'FETCH_PETS_REQUEST';
+export const FETCH_PETS_SUCCESS = 'FETCH_PETS_SUCCESS';
+export const FETCH_PETS_FAILTURE = 'FETCH_PETS_FAILTURE';
```

Comment:
Separate file with action types is not required when using `typesafe-actions`. You can use string literals directly inside action creators.

Reply:
Okay, understood: )

Reply:
Got it! Thanks ; )